src/CouchbaseLibrary.py

```diff
@@ -37,13 +37,15 @@ def connect_to_couchbase_bucket(self, host, port, bucket_name, password=None, al
         _bucket_name_ - couchbase bucket name;\n
         _password_ - password;\n
         _alias_ - connection alias;\n
+        _ipv6_ - parameter to allow ipv6 connection. Possible values: "allow", "only";\n
```

Comment:
I think you have missed the possible value `disabled` for `ipv6` parameter.

Reply:
Thanks, I fixed it.